src/views/Calculator.vue

```diff
@@ -0,0 +1,487 @@
+                <td class="font-bold text-lg -mr-6 -ml-10">
+                  {{ downpayments.percent }}%
+                </td>
+                <td class="font-medium text-base">
+                  {{
+                    $formatCurrency(
+                      Math.ceil(downpayments.actualDownpayment)
+                    ) || ""
+                  }}
+                </td>
+                <td class="font-medium text-base">
+                  {{
+                    $formatCurrency(
+                      Math.ceil(downpayments.actualRepayment / 12)
+                    ) || ""
+                  }}
+                </td>
+                <td class="font-medium text-base">
+                  {{
+                    $formatCurrency(Math.ceil(downpayments.actualRepayment)) ||
+                    ""
+                  }}
+                </td>
+                <td class="font-medium text-base">
+                  {{ $formatCurrency(Math.ceil(downpayments.total)) || "" }}
+                </td>
+              </tr>
+            </tbody>
+          </table>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import registerBg2 from "@/assets/img/register_bg_2.png";
+import AutoComplete from "@/components/Autocomplete/AutocompleteSearch.vue";
+import calculate from "../utilities/calculator";
+import { get, post } from "../utilities/api";
+import ArrowUp from "../components/svgs/arrowup.vue"
+export default {
+  data() {
+    return {
+      registerBg2,
+      apiUrls: {
+        getProduct: `/api/inventory`,
+        getCalculation: `/api/price_calculator`,
+        businessTypes: `/api/business_type`,
+        downPaymentRates: `/api/down_payment_rate`,
+      },
+      selectedProduct: {},
+      downPaymentRates: [],
+      products: [],
+      businessTypes: [],
+      product: "",
+      isActive: false,
+      repaymentCycle: 14,
+      repaymentDuration: 180,
+      calculation: [],
+      fPayment: "",
+      tpayment: "",
+      pPrice: "",
+      rPayment: "",
+      downPaymentRates: [],
+      eachDownpayment: {},
+      totals: [],
+      test1: true,
+      select_product: false,
+      downPaymentArr: [],
+      selectedDownpayment: null,
+      downpaymentCalculations: [],
+    };
+  },
+  components: {
+    AutoComplete, 
+    ArrowUp
+  },
+  computed: {
+    computedGetCalc() {
+      return this.downpaymentCalc();
+    },
+    toggleColor(){
+      let color = '';
+      if(this.selectedDownpayment == null){
+        color = 'bg-gray-300 '
+      } else if (this.selectedDownpayment.percent == 20){
+         color = 'bg-blue-300'
+      }else if (this.selectedDownpayment.percent == 40){
+          color = 'bg-red-300'
+      }else if (this.selectedDownpayment.percent == 60){
+        color = 'bg-green-300'
+      }else{
+         color ='bg-yellow-300'
+      }
+      return  color 
+    
+    },
+    toggletextColor(){
+      let color = '';
+      if(this.selectedDownpayment == null){
```

Comment:
could use switch statement here `switch (this.selectedDownpayment) {
                case 20:
                    color = 'text-blue-500' 
                    break;
            
                default:
                     color = 'text-gray-500'
                    break;
            }
            return color;`

Reply:
ok